src/hooks/gql/useVaults.ts

```diff
@@ -0,0 +1,31 @@
+import { client } from 'lib/gql/client';
+import { useQuery } from 'react-query';
+
+export function useVaults(orgId: number) {
```

Comment:
so... one of the things that `useVaults` in `src/recoilState/vaults` does is look up the actual token symbol for a simple token and set `vault.symbol` to that value. for parity, we either have to do that here as well, or look up the token symbol during creation and save it instead of "OTHER".

Reply:
I've changed introduced a symbol that's populated by the type or erc20 symbol

Reply:
Done.